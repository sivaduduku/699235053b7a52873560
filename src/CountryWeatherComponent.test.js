import React from 'react';
import { shallow } from 'enzyme';
import ContryWeatherComponent from './CountryWeatherComponent';

let wrapped = shallow(<ContryWeatherComponent />);

describe('ContryWeatherComponent', () => {
  it('should render the ContryWeatherComponent Component correctly', () => {   
    expect(wrapped).toMatchSnapshot();
  });
  it('renders the titile of the page', () => {   
    expect(wrapped.find('h1').text()).toEqual('Weather Details');
  });
});