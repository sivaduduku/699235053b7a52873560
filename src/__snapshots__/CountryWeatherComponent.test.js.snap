// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ContryWeatherComponent should render the ContryWeatherComponent Component correctly 1`] = `
<div>
  <h1>
    Weather Details
  </h1>
  <hr />
  <div>
    <div>
      temperature : 
    </div>
    <div>
      wind_speed : 
    </div>
    <div>
      precip : 
    </div>
    <div>
      weather_icons : 
      <img />
    </div>
  </div>
</div>
`;
